src/main/webapp/WEB-INF/configs/spring-datasource.xml

```diff
@@ -13,7 +13,7 @@
 		class="org.springframework.jdbc.datasource.DriverManagerDataSource">
 		<property name="driverClassName" value="${driverClassName}" />
 		<property name="url" value="${url}" />
-		<property name="username" value="${username}" />
+		<property name="username" value="${username1}" />
```

Comment:
sao lại để tên là username1, đặt lại tên khác đi

Reply:
dạ để em sửa :( em để username nó toàn nhận bừa thôi chứ k ra cái root ấy ạ

Reply:
`dạ để em sửa` > thấy sửa mô